tickit/devices/pneumatic/pneumatic.py

```diff
@@ -3,44 +3,54 @@
 from softioc import builder
 
 from tickit.adapters.epicsadapter import EpicsAdapter, InputRecord, OutputRecord
-from tickit.core.device import ConfigurableDevice, Device, DeviceUpdate
+from tickit.core.device import ConfigurableDevice, DeviceUpdate
 from tickit.core.typedefs import SimTime
 from tickit.utils.compat.typing_compat import TypedDict
 
 
 class Pneumatic(ConfigurableDevice):
+    """Pneumatic Device with movement controls."""
 
     Output = TypedDict("Output", {"output": float})
 
     def __init__(
         self,
-        name: str = "PNEUMATIC",
         initial_speed: float = 2.5,
         initial_state: bool = False,
     ) -> None:
-        self.name: str = name
+        """Initialise a Pneumatic object."""
         self.speed: float = initial_speed
         self.state: bool = initial_state
         self.moving: bool = False
         self.time_at_last_update: float = 0.0
 
     def set_speed(self, speed: float) -> None:
+        """Set the speed of movement for the device."""
         self.speed = speed
 
     def get_speed(self) -> float:
+        """Get the speed of movement of the device."""
         return self.speed
 
     def set_state(self, value) -> None:
+        """Toggles the target state of the device."""
```

Comment:
`value` should have a type hint and be included in the docstring

Reply:
value isn't actually used in this function, maybe remove the arg?

Reply:
I guess so?